raw-cms-app/src/modules/core/components/collection-item-details/collection-item-details.js

```diff
@@ -26,6 +24,45 @@ const _CollectionItemDetailsDef = async () => {
         return `/api/CRUD/${this.collectionName}`;
       },
     },
+    created: async function() {
+      this.formFields = await this.getFormFields();
+    },
+    data: function() {
+      return {
+        entitiesSchemaService: entitiesSchemaService,
+        formState: {},
+        formFields: [],
+      };
+    },
+    methods: {
+      getFormFields: async function() {
+        const res = await this.entitiesSchemaService.getPage({
+          size: 1,
+          rawQuery: { CollectionName: this.collectionName },
```

Comment:
@keggyy @marcuson we want to keep all presentation logic on presentation layer? In case it is correct to get raw collection schema info and then adapt it to the formly standard (as in this pr). Doing that, in case of extension, to manage unplanned field type, we need to extend the frontend with manay "handler" that produces item config basic on server config. Image the email plugin, that inject on client side the "handler" and on the server side the email validation. Other solution is to keep all on backend, exposing a serivice that build directly the formly json. The secon wat may be simpler, but brings presentation logic on the backend. 

I hope to have been clear, we can discuss more abut it on gitter/slack.